WIP-CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Helm upgraded to `v3.8.0`.
 - Helmfile upgraded to `v0.144.0`.
 - Helm-secrets upgraded to `v3.12.0`.
+- Node-local-dns to `0.1.1 / latest`
```

Comment:
You could note the container version here instead. Since the helm chart is our own (right?) I don't think that version is very interesting to note.